src/index.js

```diff
@@ -1,13 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
[REQUIRED] As I have stated above, Let's not disable Eslint because ESLint helps enforce best practices and coding standards, which can improve the overall quality of the codebase. This leads to more maintainable, reliable, and scalable code.
Kindly fix the errors manually before requesting another review 💪🏽

Reply:
Fixed.

Reply:
- Well resolved 👍🏽